src/OpenCV_Mat.jl

```diff
@@ -41,27 +41,35 @@ inline void ptr_val3(cv::Mat &img, int row, int col, std::vector<double> vec)
 
 function pixget(img, row::Int, col::Int)
   (row < 0 || col < 0 || row > rows(img) || col > cols(img)) ? throw(BoundsError()) : nothing
-
-  # Grayscale and binary images
-  val = @cxx ptr_val(img, row, col);
-  println(val)
-  return (val)
-
-  # RGB images
-  vec = @cxx ptr_val3(img, row, col);
-  println([int(at(vec, 0)), int(at(vec, 1)), int(at(vec, 2))])
-  return (vec)
+  image = Images.separate(img);
```

Comment:
I think it's pretty thoroughly documented at http://juliaimages.github.io/latest/. That said, if you notice stuff that's missing, by all means file an issue.

I'm not against the idea of a Wiki, but that's not where I've put my efforts thus far.